tests/integration/test_cloudformation.py

```diff
@@ -113,6 +113,30 @@ def handler(event, context):
     }
 })
 
+TEST_CHANGE_SET_BODY = """
+Parameters:
+  EnvironmentType:
+    Type: String
+    Default: local
+    AllowedValues:
+      - prod
+      - stage
+      - dev
+      - local
+
+Conditions:
+  IsProd:
+    !Equals [ !Ref EnvironmentType, prod ]
+
+Resources:
+  MyBaseDomainName:
+    Type: AWS::SSM::Parameter
+    Properties:
+      Name: !Join [ '/', [ !Ref SSMPrefix, domains, 'base-domain-name' ] ]
```

Comment:
Looks like the test is failing because `!Ref SSMPrefix` cannot be resolved. We can use a static string for the `Name` parameter instead.